tests/PHPStan/Rules/Functions/data/bug-5474.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types = 1);
+
+namespace Bug5474;
```

Comment:
If you use the same code sample multiple times, you need to define different namespace for the 2nd file (like `namespace Bug5474Two;` otherwise you have duplicate definitions of the same functions which is gonna lead to headaches :)

Reply:
oh ok, what do you prefer here - using the same code sample multiple times (copy/pasted) or requiring it from somewhere else via smth like `../Functions/data/bug-foo.php`? then I wouldn't have to care about the namespace I guess, but I didn't really find this construct I think and wasn't sure what you prefer.

Reply:
I think having the file only a single time and using `../Functions/data/bug-foo.php` is fine because it's unlikely we'll touch it again :) Because it's only used in RuleTestCase.

If we wanted to test the same problem using RuleTestCase and NodeScopeResolverTest which would mean there are some assertType calls, I'd prefer multiple files.